pipeline_0_annotate.sh

```diff
@@ -45,6 +26,60 @@ vep \
 --shift_3prime 1 \
 --no_escape \
 --numbers \
---fork 4
+--fork ${PARALLEL_THREADS}
 
 module unload VEP
+
+#CAPICE
+
+CAPICE_INPUT="${INPUT}"
+CAPICE_OUTPUT_DIR="${ANNOTATE_OUTPUT_DIR}"/step0b_capice
+
+if [[ ${OUTPUT_FILE} == *vcf ]]
+then
+    CAPICE_OUTPUT="${CAPICE_OUTPUT_DIR}"/"${OUTPUT_FILE/.vcf/.tsv}"
+    CAPICE_OUTPUT_VCF="${CAPICE_OUTPUT_DIR}"/"${OUTPUT_FILE/.vcf/.vcf.gz}"
+    CAPICE_OUTPUT_LOG="${CAPICE_OUTPUT}.log"
+else
+    CAPICE_OUTPUT="${CAPICE_OUTPUT_DIR}"/"${OUTPUT_FILE/.vcf.gz/.tsv}"
+    CAPICE_OUTPUT_VCF="${CAPICE_OUTPUT_DIR}"/"${OUTPUT_FILE}"
+    CAPICE_OUTPUT_LOG="${CAPICE_OUTPUT/.tsv/.log}"
+fi
+
+mkdir -p "${CAPICE_OUTPUT_DIR}"
+
+module load CADD
+# strip headers from input vcf for cadd
+gunzip -c $CAPICE_INPUT | sed '/^#/d' | bgzip > ${CAPICE_OUTPUT_DIR}/input_headerless.vcf.gz
+CADD.sh -a -g ${ASSEMBLY} -o ${CAPICE_OUTPUT_DIR}/cadd.tsv.gz ${CAPICE_OUTPUT_DIR}/input_headerless.vcf.gz
+module unload CADD
+
+module load CAPICE
+python ${EBROOTCAPICE}/CAPICE_scripts/model_inference.py \
+--input_path ${CAPICE_OUTPUT_DIR}/cadd.tsv.gz \
+--model_path ${EBROOTCAPICE}/CAPICE_model/xgb_booster.pickle.dat \
+--prediction_savepath ${CAPICE_OUTPUT} \
+--log_path ${CAPICE_OUTPUT_LOG}
+
+module load Java
+java -Djava.io.tmpdir="${TMPDIR}" -XX:ParallelGCThreads=2 -Xmx1g -jar capice2vcf.jar -i ${CAPICE_OUTPUT} -o ${CAPICE_OUTPUT_VCF}
+module unload Java
+
+#VcfAnno
+
+VCFANNO_INPUT="${VEP_OUTPUT}"
+VCFANNO_OUTPUT_DIR="${ANNOTATE_OUTPUT_DIR}"/step0c_vcfAnno
+VCFANNO_OUTPUT="${VCFANNO_OUTPUT_DIR}"/"${OUTPUT_FILE}"
+VCFANNO_OUTPUT_LOG="${VCFANNO_OUTPUT}.log"
+
+mkdir -p "${VCFANNO_OUTPUT_DIR}"
+
+module load vcfanno
+module load HTSlib
+#inject location of the capice2vcf tool in the vcfAnno config.
+CAPICE_OUTPUT_fixed="${CAPICE_OUTPUT_VCF/\.\//}"
```

Comment:
lets stick to the convention to uppercase all bash variable names: CAPICE_OUTPUT_fixed --> CAPICE_OUTPUT_FIXED